<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Daraz Wishlist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/resource/image/mansaranna.png</normaloff>:/images/resource/image/mansaranna.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	padding:10px;
	
	background-color: rgba(255, 255, 255, 100);
	border-radius: 10px;
	color: rgb(0, 0, 0);
}

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <property name="styleSheet">
        <string notr="true">#mainPage{
	
	border-image: url(:/images/resource/image/mansaranna.png);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loginBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>58</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
	color: rgb(255, 255, 255);
}

QPushButton{
	padding:8px;
	background-color: rgba(255, 255, 255, 100);
	border-radius:25px;
}

QPushButton:hover{
	padding:8px;
	
	background-color: rgb(42, 42, 42);
	border-radius:25px;
}</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/darazLogo.png</normaloff>:/images/darazLogo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wishlistPage">
       <property name="styleSheet">
        <string notr="true">#wishlistPage{
	background-color: rgb(71, 71, 71);
}
QPushButton{
	padding:8px;
		color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 100);
	border-radius:25px;
}

QPushButton:hover{
	padding:8px;
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255, 100);
	border-radius:25px;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" alignment="Qt::AlignLeft">
            <widget class="QPushButton" name="back1Btn">
             <property name="styleSheet">
              <string notr="true">QPushButton{
	padding:5px;
	background-color: rgba(255, 255, 255, 0);

}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/resource/icon/chevron-left.svg</normaloff>:/icons/resource/icon/chevron-left.svg</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="2" alignment="Qt::AlignRight">
            <widget class="QPushButton" name="settingBtn">
             <property name="styleSheet">
              <string notr="true">QPushButton{
	padding:10px;
	background-color: rgba(255, 255, 255, 0);
	border-radius:25px;
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/resource/icon/settings.svg</normaloff>:/icons/resource/icon/settings.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QTextEdit" name="textEdit">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#textEdit{
	
	color: rgb(0, 0, 0);
	border-radius:25px;
	background-color: rgba(255, 255, 255, 150);
}</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="addWishListBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>ADD to Wishlist</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/images/resource/image/wishlist.png</normaloff>:/images/resource/image/wishlist.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="addWishListStopBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>434</width>
               <height>52</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 82, 82,200);</string>
             </property>
             <property name="text">
              <string>STOP</string>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingPage">
       <property name="styleSheet">
        <string notr="true">#settingPage{
		background-color: rgb(71, 71, 71);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="back2Btn">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	padding:5px;
	background-color: rgba(255, 255, 255, 0);

}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/resource/icon/chevron-left.svg</normaloff>:/icons/resource/icon/chevron-left.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>746</width>
            <height>630</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/images/resource/image/help.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="wishlistIDSaveBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(93, 255, 112);
	color: rgb(0, 0, 0);
	padding:5px;
	border-radius:10px;
}

QPushButton:hover{
	background-color: rgb(71, 255, 30);
}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Wishlist Button
 ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_wishlistID">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Ex: block-CATyli37LyV</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>All Right Reserved @ 2023</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addWishListBtn</sender>
   <signal>clicked()</signal>
   <receiver>addWishListStopBtn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>758</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>771</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addWishListStopBtn</sender>
   <signal>clicked()</signal>
   <receiver>addWishListBtn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>782</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>790</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addWishListStopBtn</sender>
   <signal>clicked()</signal>
   <receiver>addWishListStopBtn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>766</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addWishListBtn</sender>
   <signal>clicked()</signal>
   <receiver>addWishListBtn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>767</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
